_projects/0050_smeqa.md

```diff
@@ -0,0 +1,63 @@
+---
+# Page template info (DO NOT EDIT)
+layout: default
+banner_file: banner--people-projects-lg.svg
+banner_file_mobile: banner--people-projects-sm.svg
+project_page: true
+
+# Carousel (Edit this)
+carousel_title: "Changing how the government hires"
+carousel_summary: "We helped develop a process that allows HR to leverage subject matter experts to evaluate candidates for specialized roles. The result restores fair and open access for all applicants, shortens the hiring timeline, and ensures applicants are truly qualified."
+carousel_image_name: project-smeqa-card.jpg
+
+# Project detail page (Edit this)
+title: "Subject Matter Expert Qualification Assessments (SME‑QA)"
+agency: Cross-agency
+permalink: projects/smeqa
+project_url: 
+
+# Impact statement (Edit this)
+impact_statement:
+  - figure: "45 --> 16"
```

Comment:
```suggestion
  - figure: "45 &rarr; 16"
```

effect of the suggested change: 
<img src="https://user-images.githubusercontent.com/31673962/98850463-d373c600-2422-11eb-8823-d5b008712557.png" width=100>   <img src="https://user-images.githubusercontent.com/31673962/98850475-d7074d00-2422-11eb-996a-50c1139b48a7.png" width=100>